src/govuk/helpers/typography.test.js

```diff
@@ -419,4 +417,145 @@ describe('@mixin govuk-typography-responsive', () => {
               line-height: 1.42857; } }`)
     })
   })
+
+  describe('@mixin govuk-font', () => {
+    it('outputs all required typographic CSS properties', async () => {
+      const sass = `
+      // Avoid font face being output in tests
+      $govuk-include-default-font-face: false;
+      ${sassBootstrap}
+
+      .foo {
+        @include govuk-font($size: 14)
+      }`
+
+      const results = await renderSass({ data: sass, ...sassConfig })
+
+      expect(results.css.toString().trim()).toBe(outdent`
```

Comment:
This is basically a 'snapshot test', but I couldn't think of a better way of doing it… this does mean that whenever anything changes in the govuk-font mixin we'll need to update this test accordingly. It's not ideal, but I think that's OK?